aten/src/ATen/native/cpu/VectorizedCopy.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/cpu/vec256/vec256.h>
+
+namespace at {
+namespace native {
+
+template <typename T>
+using VecType = at::vec256::Vec256<T>;
+// Copies num_elements elements from src to dest for total of repeat_factor times.
+template <typename DataType>
+inline void single_element_broadcast(DataType* dest_ptr, const DataType src_element, const uint64_t repeat_factor) {
+  auto src_broadcasted = at::vec256::Vec256<DataType>(src_element);
+  auto size = at::vec256::Vec256<DataType>::size();
+  for (uint64_t j = 0; j < repeat_factor / size; ++j) {
+    src_broadcasted.store(dest_ptr);
+    dest_ptr += size;
+  }
+  for (uint64_t j = 0; j < repeat_factor % size; ++j) {
+    *dest_ptr = src_element;
+    ++dest_ptr;
+  }
+}
+
+template <typename DataType>
+void vectorized_copy(DataType* dest_ptr, const DataType* src_ptr, const uint64_t num_elements) {
```

Comment:
This function seems unnecessary

Reply:
Sure.

Reply:
A vectorized copy is being worked on separately. cc @VitalyFedyunin